### ISSUE TITLE
How to use different base network

### ISSUE THREAD

**Issue by achintacs90** [RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras#19]
Sir , i want to use different base network, can you please tell me the procedure 
I know you use transfer learning , but can i use pre train weights of a simple image classification model

**Comment by StephenRUK** [comment]
Hi, you should look at the author's article on Medium and the notebook for training. You can replace the base network with your own easily, see `nn_base` in the training notebook.

**Comment by achintacs90** [comment]
Can you give the original author github link 
Previous link is not working now

**Comment by StephenRUK** [comment]
You can find the notebook in this repository. https://github.com/RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras/blob/master/frcnn_train_vgg.ipynb

**Comment by achintacs90** [comment]
Thank you 
I can replace base network easily , can you tell the procedure for generating the pre train weights for that base network

**Comment by StephenRUK** [comment]
You can use pretrained weights for common models e.g. VGG16, ResNet or such. See the Keras docs for available models with ImageNet weights, if you wish to use them https://keras.io/applications/

**Comment by achintacs90** [comment]
Can i use pre train weights of  other than these network

**Comment by StephenRUK** [comment]
You can use any network you want. I think you need to do some research in general about transfer learning or finetuning of existing networks. It is not specific to this model or GitHub repository.

**Comment by achintacs90** [comment]
can i train whole network without using transfer learning 
please reply

**Comment by StephenRUK** [comment]
Yes you can train the whole network, but it makes no sense to train a whole VGG or ResNet model from scratch. This isn't a support forum & I'm not even part of this repo 😉 do check out an article such as this https://www.learnopencv.com/keras-tutorial-transfer-learning-using-pre-trained-models/

**Comment by achintacs90** [comment]
Thank you sir

**Comment by DTW1004** [comment]
![image](https://user-images.githubusercontent.com/43997072/56503414-f3dde600-6547-11e9-8519-3549a3d55af6.png)
 can you please tell me how to get these four file for train?

**Comment by devendraswamy** [comment]
> ![image](https://user-images.githubusercontent.com/43997072/56503414-f3dde600-6547-11e9-8519-3549a3d55af6.png)
> can you please tell me how to get these four file for train?

you just create a "model" folder in base_path then reaming files are automatically stored
